_data/cryptocurrencies.yml

```diff
@@ -28,6 +28,13 @@ websites:
       tfa: Yes
       software: Yes
 
+ - name: Bitfinex
+      url: https://www.bitfinex.com
+      img: bitfinex.png
+      tfa: Yes
+      software: Yes
+      doc: https://support.bitfinex.com/hc/en-us/sections/203192029-Security
+
```

Comment:
I noticed that their docs page states that they also support SMS? If that's the case, perhaps add it.